frontend/src/views/AboutView.vue

```diff
@@ -1,65 +1,127 @@
+                  ></v-btn>
+                </v-card-actions>
+              </v-card>
+            </v-col>
+          </v-row>
+        </v-container>
+      </div>
+
+      <div>
+        <h3>Useful Links:</h3>
+        <v-list>
+          <v-list-item
+            title="Interactive Hydrologic Process Taxonomy:"
+            class="mb-5"
+          >
+            <a
+              href="http://mcmillanhydrology.org/ProcessTaxonomy/ProcessTaxonomyDiagram.html"
+            >
+              http://mcmillanhydrology.org/ProcessTaxonomy/ProcessTaxonomyDiagram.html
+            </a>
+          </v-list-item>
+
+          <v-list-item
+            title="Journal Articles Describing this Project:"
+            subtitle="McMillan, H., Araki, R., Gnann, S., Woods, R., & Wagener, T.
+                (2023)."
+          >
+            <div>
+              How do hydrologists perceive watersheds? A survey and analysis of
+              perceptual model figures for experimental watersheds.
+            </div>
+            <a href="https://onlinelibrary.wiley.com/doi/full/10.1002/hyp.14845"
+              >Read the Article
+            </a>
+          </v-list-item>
+        </v-list>
+      </div>
+    </div>
   </v-container>
 </template>
 
-<style>
-@media (min-width: 1024px) {
-  .about {
-    min-height: 100vh;
-  }
-}
-</style>
-
 <script setup>
+import { mdiOpenInNew } from '@mdi/js';
 
 const members = [
-  {
-    name: 'Scott Black',
-    image: 'https://www.cuahsi.org/uploads/team/img/_headshot/black-scott.jpg',
-    position: 'Senior Software Specialist',
-    org: 'CUAHSI'
-  },
-  {
-    name: 'Devin Cowan',
-    image: 'https://www.cuahsi.org/uploads/team/img/_headshot/Devin-Cowan.jpg',
-    position: 'Research Programmer',
-    org: 'CUAHSI'
-  },
-  {
-    name: 'Irene Garousi-Nejad',
-    image: 'https://www.cuahsi.org/uploads/team/img/_headshot/Irene_headshot03.jpg',
-    position: 'Research Scientist',
-    org: 'CUAHSI'
-  },
-  {
-    name: 'Tony Castronova',
-    image: 'https://www.cuahsi.org/uploads/team/img/_headshot/TCastronova.jpg',
-    position: 'Senior Research Hydrologist',
-    org: 'CUAHSI'
-  },
-]
+        {
+          name: 'Hilary McMillan',
+          designation: 'Professor & Project Lead, San Diego State University',
+          img: 'https://lh7-rt.googleusercontent.com/docsz/AD_4nXdADheKsAYX2k0qqlrcKs1K045hXWzS2HBmPrFg9CFbIpcw5TKhCq6ufU6t8T5J7t9ZLpPzfCtzNgUXnDp-4swbkik01dvRoX_lPI09jbCHpbi6eFmkHbry9agRxKI1tC9ePF8pKpCumi5EXAXIwV3iXV9T?key=dlmBhZGLWn-vOON8gSsrvg',
+          link: 'http://www.mcmillanhydrology.org/',
+        },
+        {
+          name: 'Ryoko Araki',
+          designation: 'PhD Student & Developer, San Diego State University',
+          img: 'https://lh7-rt.googleusercontent.com/docsz/AD_4nXc_qLv2wfCPet7DY4CVIUk-q-6fAn5ySN5UKXIrXs4J8_F1OObBotovTMbgGHvdTWwRKLOpsUJvF275M-61mHUtL0IFvyW15VDQJq_pyeR8E1SC0szlcGwYzqmoydj5ryHXihvSTtqhKbSgWl98HEZXPMbf?key=dlmBhZGLWn-vOON8gSsrvg',
```

Comment:
I get a [429](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/429) response for these images:
![image](https://github.com/user-attachments/assets/79232fbd-cffa-4fd0-93b1-18e64bf0bab8)

Why are we hosting the in googleusercontent?
If possible, I would either embed these directly into our build, or host them on a more reliable location

Reply:
Updated the code @devincowan. Please review once